src/libs/cozyAppBundle/cozyAppBundle.ts

```diff
@@ -0,0 +1,268 @@
+import Gzip from '@fengweichong/react-native-gzip'
+import RNFS from 'react-native-fs'
+import { logger } from '/libs/functions/logger'
+
+import {
+  fetchCozyAppArchiveInfoForVersion,
+  fetchCozyAppVersion,
+  getFqdnFromClient
+} from '/libs/client'
+import { getBaseFolderForFqdnAndSlug } from '/libs/httpserver/httpPaths'
+import {
+  getCurrentAppConfigurationForFqdnAndSlug,
+  setCurrentAppVersionForFqdnAndSlug
+} from '/libs/cozyAppBundle/cozyAppBundleConfiguration'
+import CozyClient from 'cozy-client'
+import { getErrorMessage } from '/libs/functions/getErrorMessage'
+import {
+  getVersionsToKeep,
+  handleCleanup
+} from '/libs/cozyAppBundle/cozyAppBundle.functions'
+
+export const log = logger('AppBundle')
+
+const BUNDLE_UPDATE_DELAY_IN_MS = 10000
+
+export type AppType = 'konnectors' | 'apps'
+
+export interface AppInfo {
+  client: CozyClient
+  slug: string
+}
+
+/**
+ * After the specified delay, check the cozy-app version on cozy-stack and
+ * update the local CozyAppBundle if necessary
+ *
+ * This method should run in background and cannot be awaited
+ */
+export const updateCozyAppBundleInBackground = ({
```

Comment:
⚠️  As you removed the JSDoc, we lost the mention to default value and so it won't appear in the autocomplete documentation.
This is not really a problem as this is an internal method, but we should find a way to document TS code correctly in the future.

Reply:
Agree